From 4a5b355ae11f4ef987c5714a5769abd7b529777a Mon Sep 17 00:00:00 2001
From: Nathan Chancellor <nathan@kernel.org>
Date: Tue, 7 Sep 2021 16:41:55 -0700
Subject: [PATCH 1/6] HACK: LONGTERM: Disable -Wframe-larger-than= for clang

There are a lot of these warnings right now so disable the warning for
now until they can be triaged and fixed.

Link: https://github.com/ClangBuiltLinux/linux/labels/-Wframe-larger-than%3D
Signed-off-by: Nathan Chancellor <nathan@kernel.org>
---
 lib/Kconfig.debug | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/lib/Kconfig.debug b/lib/Kconfig.debug
index 1af972a92d06..e494e8b812f8 100644
--- a/lib/Kconfig.debug
+++ b/lib/Kconfig.debug
@@ -439,8 +439,9 @@ config GDB_SCRIPTS
 endif # DEBUG_INFO
 
 config FRAME_WARN
-	int "Warn for stack frames larger than"
+	int "Warn for stack frames larger than" if !CC_IS_CLANG
 	range 0 8192
+	default 0 if CC_IS_CLANG
 	default 0 if KMSAN
 	default 2048 if GCC_PLUGIN_LATENT_ENTROPY
 	default 2048 if PARISC

base-commit: ff202c5028a195c07b16e1a2fbb8ca6b7ba11a1c
-- 
2.48.1

